setup.cfg

```diff
@@ -33,6 +33,7 @@ exclude =
     env  # exclude asv benchmark environments from linting
 
 [flake8-rst]
+max-line-length = 88
```

Comment:
As mentioned on the other PR as well, I think this is too much, because that doesn't fit on the rendered html doc page. Based on one example I looked at in the other PR, something like 84 might be better.